.github/workflows/cleanup.yml

```diff
@@ -0,0 +1,13 @@
+name: 'Artifacts cleanup'
+on:
+  schedule:
+    - cron: '0 1 * * *' # every night at 1 am UTC
+
+jobs:
+  delete-artifacts:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: kolpav/purge-artifacts-action@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          expire-in: 7days # Set this to 0 to delete all artifacts
```

Comment:
@tjhei?

Reply:
why 7 days and not something like 14?

Reply:
I guess everything below 90 is fine. So why not 14 ;) Out of curiosity: how much volume are the artifact consuming right now? You can check that if you go to the organization and go to `Billing and plans`.

Reply:
90 seems too much right now. I think we have reached the maximum allowable space with our windows artifacts (these are roughly 400MB for each successfull build). I would have thought that 7 days was enough, but if you think we need more, let's do 14.

Reply:
> with our windows artifacts (these are roughly 400MB for each successfull build)

Wait a second. What is the purpose of storing that as artifact!?

Reply:
> I guess everything below 90 is fine. So why not 14 ;) Out of curiosity: how much volume are the artifact consuming right now? You can check that if you go to the organization and go to Billing and plans.

@tjhei  Now, I am curious how much data has been accumulated. Terabytes?

Reply:
The windows artifact is a full working installation of deal.II, that can be used as a binary installation package. Pretty much like our docker images of the master branch, but for windows. These are used by a few guys around the globe to keep their windows apps up to date.